### PR TITLE
A new start

### PR DESCRIPTION
**Pull request by dqsully**
Update the homepage of hjson.org to reflect Hjson's new status.

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -3,21 +3,12 @@
...
+We are also looking for maintainers for our [.NET](https://github.com/hjson/hjson-cs), [Go](https://github.com/hjson/hjson-go), [Java](https://github.com/hjson/hjson-java), [Python](https://github.com/hjson/hjson-py) and [Ruby](https://github.com/hjson/hjson-rb) packages. If you like Hjson and want to keep it alive, please consider helping out.
 
 ```
-Goodbye Hjson!
-
-This prjoect has reached EOL.
-You are welcome to continue
-using it, just don't expect
-any updates.
+Hjson is back, baby!
```
Comment (sffc) [inline pages/index.md]: Back from what?  The EOL announcement was only on for a couple of days over the weekend.  This message is out of context.

File: pages/index.md [hunk 2]
```diff
@@ -3,21 +3,12 @@
...
 
-When I started this project in 2014 I was aiming to deliver a better experience when managing configuration files in JSON and other formats like YAML.
+This project is now entirely community-supported. If you would like to help or want to report an issue, please see our [GitHub](https://github.com/hjson/).
 
-But now it's time to admid that Hjson has failed to be picked up by any major projects and this is unlikely to change.
-
-Hjson is stable and you are welcome to do with it as you may wish, but because my time is limited I will no longer be updating the spec, the implementations or the syntax definitions.
-
-I would like to thank all contributors for their help! While the project was not as successful as it deserved to be, it was a great learning experience in every way.
+We are also looking for maintainers for our [.NET](https://github.com/hjson/hjson-cs), [Go](https://github.com/hjson/hjson-go), [Java](https://github.com/hjson/hjson-java), [Python](https://github.com/hjson/hjson-py) and [Ruby](https://github.com/hjson/hjson-rb) packages. If you like Hjson and want to keep it alive, please consider helping out.
```
Comment (sffc) [inline pages/index.md]: I suggest taking out "and want to keep it alive".  It makes it sound too ominous, like Hjson is going to die.  I don't want to give off that impression.

File: pages/index.md (hunk 1 above)
Comment (dqsully) [reply pages/index.md]: Yeah, I knew that was going to be weird, I just didn't know what to put in there.

File: pages/index.md (hunk 2 above)
Comment (dqsully) [reply pages/index.md]: Good point.

**Comment by dqsully** [comment]
I still don't feel great about those two sections, but I think they are better. I honestly have no idea what to put in the code section, and I could foresee issues with this version of it too.

Also, I found out that the code sections create the second column. I'll look into doing a second column without code tomorrow.

**Comment by sffc** [comment]
The code block didn't exist until laktak added it in 10e8ba5.  I think you can just delete it.

**Pull request merged**